<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="welcome-PC" ignored="0" name="Test Assertion" tests="2" failures="1" timestamp="06 May 2019 21:05:38 GMT" time="18.186" errors="0">
  <testcase name="getTitle" time="0.028" classname="testNG.TestNGAssertion"/>
  <testcase name="testEquality" time="0.003" classname="testNG.TestNGAssertion">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testNG.TestNGAssertion.testEquality(TestNGAssertion.java:33)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testEquality -->
</testsuite> <!-- Test Assertion -->
